.prettierignore

```diff
@@ -15,6 +15,8 @@
 **/*.sln
 **/*.txt
 **/*.xml
+**/*.key
+**/*.md
```

Comment:
Why are you ignoring markdown? If I am not wrong, Prettier supports it: https://prettier.io/blog/2017/11/07/1.8.0.html#markdown-support

Reply:
I'm getting an error with that file because it was not a root level.
Seems that prettier looks for the git directory when validating MD files. Tried using `yarn --prettier-fix` but it does nothing

Reply:
Fixed, thanks @andresmoschini